docs/README.rst

```diff
@@ -0,0 +1,156 @@
+.. _readme:
+
+sysstat-formula
+================
```

Comment:
All of the heading symbols in this file should be the same length as the heading text (i.e. also a number of the `^` heading symbols further down).